letsencrypt/storage.py

```diff
@@ -7,12 +7,17 @@
 import parsedatetime
 import pytz
 
+import logging
```

Comment:
nit: This line should be moved between `import datetime` and `import os`.

The general pattern for this repo is:

```
standard library imports

dependency imports

LE code imports
```
Each of the three groups should be in alphabetical order. Sometimes our own packages are further split up if there is a lot of imports. See `letsencrypt/client.py` for an example of that.